packages/shared/src/modules/categories/libs/types/get-categories-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type CategoryDto } from "./category-dto.type.js";
+
+type GetCategoriesDto = { items: CategoryDto[] };
```

Comment:
```suggestion
type CategoriesGetAllResponseDto = { items: CategoryDto[] };
```
Don't forget about the file name